guides/v2.2/cloud/live/sens-data-initial.md

```diff
@@ -120,6 +130,7 @@ The following snippet from `config.php` shows an example of changing the default
 
      ... more ...
 ```
+
```

Comment:
Does no-copy still work when you add a line? I didn’t realize...

Reply:
Agh... no it does not. I thought it might since the collapsible tag works with a space, but I got lazy and neglected to test first. Thank you for catching. This will br a problem for rule MD031 (blanks around fences),

Reply:
We need at least one way to turn off the copy ability. I'm open to other solutions.

Reply:
I agree. Rule MD031 is relatively unimportant, so we may need to ignore it.